src/components/images/svg/layout-thumbnails/image-paragraph.tsx

```diff
@@ -0,0 +1,44 @@
+import { SvgIcon, SvgIconProps, styled } from "@mui/material";
+
+/**
+ * Interface representing component properties
+ */
+interface Props extends SvgIconProps {}
+
+/**
+ * Styled SVGIcon container
+ */
+const Root = styled(SvgIcon, {
+  label: "custom-icon-root"
+})(({ theme }) => ({
```

Comment:
In all these image files we are getting a warning about the theme being unused, but I'm guessing it needs to be here?